src/mlpack/methods/dbscan/dbscan_main.cpp

```diff
@@ -123,6 +123,14 @@ static void mlpackMain()
       "hilbert-r", "r-plus", "r-plus-plus", "ball" }, true,
       "unknown tree type");
 
+  // Value of epsilon sohould be greater than 0.
+  RequireParamValue<double>("epsilon", [](double x) { return x > 0; },
+      true, "invalid value of epsilon specified");
```

Comment:
I think it would me more reasonable to write something like `Value of epsilon can't be negative `. Same with `min_size. `What do you think?

Reply:
I would actually say `value of epsilon must be positive`, since we are not allowing `0` as a value.  (The same applies for `min_size`).  It is a subtle difference but still a difference... :)

Reply:
I've removed 0 as it was not required. Hope the comments are fine now